### PR TITLE
[expanse-272] Expanse Expander - partial response bugfix

### PR DESCRIPTION
**Pull request by andrew-expanse**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.
- Shortened the period of time that tokens are considered valid to avoid auth errors.
- Fixes an issue related to the `ip` command where an error is generated if the API returns a partial response (which is likely in some use cases). 
- Adds friendly values for various empty fields returned by the `domain` command.
- Updated several incorrect types.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [x] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation 

## Demisto Partner?
- [x] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Comment by andrew-expanse** [comment]
> please change all the `N/A` and `Unknown` to `None`

Done - I assumed you meant the string "None" and not the keyword.

**Comment by anara123** [comment]
@andrew-expanse sorry, my bad. I should have be more specific. I mean keyword None.
If value not exists, then don't set it at all or set the value to be None.
This way is easier later to work with this data in the playbook

File: Packs/Expanse/Integrations/Expanse/Expanse.py [hunk 1]
```diff
@@ -168,25 +168,29 @@ def parse_events(events):
...
-        "Description": "None",
+        "Location": None,
+        "Country": None,
+        "Description": None,
     }
     if len(data.get('locationInformation', [])) > 0:
         geo["Location"] = "{0}:{1}".format(
-            data['locationInformation'][0].get('geolocation', {}).get('latitude', "None"),
-            data['locationInformation'][0].get('geolocation', {}).get('longitude', "None")
+            data['locationInformation'][0].get('geolocation', {}).get('latitude'),
```
Comment (anara123) [inline Packs/Expanse/Integrations/Expanse/Expanse.py]: make sure that here it can't be `None:None` because of `{0}:{1}`

**Comment by anara123** [comment]
@andrew-expanse can you add a unit test with mock data that will validate the changes you did in this PR.
Like with some fields empty or None

**Comment by anara123** [comment]
@kirbles19 please review

**Comment by kirbles19** [comment]
@anara123 Done.

**Pull request merged**